### PR TITLE
init base project

### PR DESCRIPTION
**Pull request by anhmv-1205**
### [Redmine link]
https://edu-redmine.sun-asterisk.vn/issues/13040

### CODE REVIEW

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/java/com/sun_asterisk/moviedb_44/units/BindingUtils.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import android.graphics.Bitmap
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+
+object BindingUtils {
+    private const val THUMBNAIL_SIZE_MULTIPLIER = 0.1f
+
+    @BindingAdapter("android:src")
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/units/BindingUtils.kt]:
```suggestion
    @BindingAdapter("adapter")
```

File: app/src/main/java/com/sun_asterisk/moviedb_44/data/model/Movie.kt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package com.sun_asterisk.moviedb_44.data.model
+
+class Movie
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/data/model/Movie.kt]: remove class

**Comment by anhmv-1205** [comment]
ready

**Pull request merged**